packages/FormElement/src/components/Content/renderFunctions.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+
+const cloneWithAriaAttributes = ({ element, idForLabel, ariaDescribedBy }) =>
+  React.cloneElement(element, {
+    "aria-describedby": ariaDescribedBy,
+    id: idForLabel,
+  });
+
+const getAccessibleElement = (elements, cloneFn) =>
+  Array.isArray(elements) ? elements.map(cloneFn) : cloneFn(elements);
+
+export default {
+  Input: ({ extractedElements, idForLabel, ariaDescribedBy }) =>
+    getAccessibleElement(extractedElements, (element, index = 0) =>
+      cloneWithAriaAttributes({ element, idForLabel: index === 0 ? idForLabel : null, ariaDescribedBy })
```

Comment:
This behaviour of the first Input being provided the `id` (that is associated with the formElement label) will be able to be overridden in a follow up pr

Also on line https://github.com/acl-services/paprika/pull/663/files#diff-1a53d8778ef9b8807972d1aa4463c8c9R42